lib/core/auth/bloc/auth_bloc.dart

```diff
@@ -184,8 +188,229 @@ class AuthBloc extends Bloc<AuthEvent, AuthState> {
       }
     });
 
+    /// Log in with OAuth Provider to get a code.
+    on<OAuthLoginAttempt>((event, emit) async {
+      LemmyClient lemmyClient = LemmyClient.instance;
+      String originalBaseUrl = lemmyClient.lemmyApiV3.host;
+      String instance = event.instance ?? state.oauthInstance!;
+      ProviderView provider = event.provider ?? state.oauthProvider!;
+
+      try {
+        emit(state.copyWith(status: AuthStatus.loading, account: null, isLoggedIn: false));
+
+        if (instance.startsWith('https://')) instance = instance.replaceAll('https://', '');
```

Comment:
You can do `replaceFirst` here to make sure you only replace it at the beginning (not that a scheme should ever appear elsewhere in a URL!).

Reply:
updated, thanks :)